deploy/clowdapp.yaml

```diff
@@ -104,6 +108,11 @@ parameters:
   displayName: Schedule for Unavailable Sources
   name: SCHEDULE_UNAVAILABLE
   value: "15,45 */1 * * *"
+- description: Skips fetching sources that are considered empty and for which availability checks are unnecessary to run.
+  displayName: Skip fetching empty sources.
+  name: SKIP_EMPTY_SOURCES
+  value: "false",
```

Comment:
Is that an extra ',' at the end?  I'm not sure if that will cause a problem or not?  :thinking:

Reply:
Nice catch! Removed!